UIComponets/FoodAppScreen/CustomViews/TableViewSectionHeader.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  TableViewSectionHeader.swift
+//  UIComponets
+//
+//  Created by Rajan Patel on 05/06/23.
+//
+
+import UIKit
+
+protocol LoadSecondVC: AnyObject {
+    func seeAllButtonClicked()
```

Comment:
add Delegate in protocol name where it's use for Delegate, this name misinterpreted to view controller insted of protocol